scripts/test_python_version.py

```diff
@@ -0,0 +1,39 @@
+#!/bin/env python
+
+import pathlib as pl
+import platform
+import sys
+
+
+def main():
+    major_v, minor_v, patch_v = platform.python_version_tuple()
+
+    print(f"Found python version: {major_v}.{minor_v}.{patch_v}")
+
+    min_major_v, min_minor_v = to_version(
+        (pl.Path(__file__).parent.parent / "requirements" / "PYTHON_VERSION")
+        .read_text()
+        .strip()
+    )
+
+    exit_code = (
+        1
+        if int(major_v) < min_major_v
+        or (int(major_v) == min_major_v and int(minor_v) < min_minor_v)
+        else 0
```

Comment:
you can use packaging.Version which understands all this and then just do something like
```python
from packaging import Version
import sys
version_found = Version(sys.version)
version_expected = Version((Path(__file__).parent.parent / "requirements" / "PYTHON_VERSION").read_text())
if version_found < version_expected:
    sys.exit(1)
```

Reply:
True. And if you want to be more readable a small tip
```python
import os
from packaging import Version
import sys
from pathlib import Path

repo_base = Path(__file__).parent.parent

if Version(sys.version) < Version((repo_base / "requirements" / "PYTHON_VERSION").read_text()):
     sys.exist(os.EX_DATAERR)
```

Reply:
Thx for the suggestion, but i think I'll leave it like it is, since afaik packaging is an external package. Since this script will possibly be run on machines that don't have this installed, i want to make it as compact as possible.

I was even thinking about removing the f-strings, because it's kind of useless to check for python>2 because these f-strings will crash that script anyway on older pythons :-)

Reply:
we do not support below 3.10 anyway (exception for the director-v0, which is fixed in 3.6)